From c2f78b39942a408918a4453e87a3925f62b2dcb2 Mon Sep 17 00:00:00 2001
From: chromium-autoroll
 <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Date: Mon, 25 Feb 2019 07:23:07 +0000
Subject: [PATCH] Roll src/third_party/skia e1636a4b56aa..5bbd02d6bb3d (1
 commits)

https://skia.googlesource.com/skia.git/+log/e1636a4b56aa..5bbd02d6bb3d


git log e1636a4b56aa..5bbd02d6bb3d --date=short --no-merges --format='%ad %ae %s'
2019-02-25 skia-recreate-skps@skia-swarming-bots.iam.gserviceaccount.com Update go_deps asset


Created with:
  gclient setdep -r src/third_party/skia@5bbd02d6bb3d

The AutoRoll server is located here: https://autoroll.skia.org/r/skia-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff, who should
be CC'd on the roll, and stop the roller if necessary.

CQ_INCLUDE_TRYBOTS=luci.chromium.try:android_optional_gpu_tests_rel;luci.chromium.try:linux-blink-rel;luci.chromium.try:linux-chromeos-compile-dbg;luci.chromium.try:linux_optional_gpu_tests_rel;luci.chromium.try:mac_optional_gpu_tests_rel;luci.chromium.try:win_optional_gpu_tests_rel
TBR=scroggo@chromium.org

Change-Id: I8b9fc0f7f9f866bf1a33a6420a4847ed7f101226
Reviewed-on: https://chromium-review.googlesource.com/c/1484740
Reviewed-by: chromium-autoroll <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Commit-Queue: chromium-autoroll <chromium-autoroll@skia-public.iam.gserviceaccount.com>
Cr-Commit-Position: refs/heads/master@{#635041}
---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index 43cb129b5ce0a..ab587ea7d634c 100644
--- a/DEPS
+++ b/DEPS
@@ -127,7 +127,7 @@ vars = {
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling Skia
   # and whatever else without interference from each other.
-  'skia_revision': 'e1636a4b56aacd0e9f26bf7d0dd2a2f14afe2bd3',
+  'skia_revision': '5bbd02d6bb3d933fbe73d6b8fc915caa120bbcaf',
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling V8
   # and whatever else without interference from each other.
